packages/manager/modules/nasha/src/components/edit-name/edit-name.controller.js

```diff
@@ -0,0 +1,38 @@
+export default class NashaComponentsEditNameController {
+  /* @ngInject */
+  constructor($scope, $translate, $http) {
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.$http = $http;
+    this.isSubmitting = false;
+    this.name = '';
+  }
+
+  get baseName() {
+    return this.nasha?.customName || '';
+  }
+
+  get canSubmit() {
+    return (
+      (this.$scope.editNameForm?.$valid && this.name !== this.baseName) || false
```

Comment:
Question: do you need "|| false" in the condition? the first part evaluates to true or false right?